docker/Dockerfile

```diff
@@ -79,6 +79,7 @@ RUN set -x \
   " \
   && apt-get update \
   && apt-get install -y --no-install-recommends $buildDeps \
+  && python -m pip install --upgrade 'pip>=20.3.0' \
```

Comment:
If we use exact pinning we would have an idempotent step. Would that be preferred?

Reply:
I'm hoping to remove this step once the base `python-3.6` images start using the latest PIP. If that does not happen, I'll convert this to an exact one.